src/Catalog/PackageDeprecationItem.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Versioning;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public class PackageDeprecationItem
+    {
+        public PackageDeprecationItem(
+            IReadOnlyCollection<string> reasons,
+            string message = null,
```

Comment:
I think all of these parameters should be explicit since they are all known at the same time from DB, i.e. no defaults are necessary.

Reply:
done